bionic/tokenization.py

```diff
@@ -53,6 +55,7 @@ def clean_str(string):
 HASH_LEN = 5
 
 
+# to-do: add optional directory parameter for where to write/read from
```

Comment:
nit: the convention in the rest of the project is `TODO`.  The only reason I care is that it makes it easier to search for all the outstanding `TODO`s if we use the same convention everywhere.

Reply:
Updated!